actix-identity/src/identity_errors.rs

```diff
@@ -0,0 +1,69 @@
+use actix_session::{SessionGetError, SessionInsertError};
+use actix_web::{cookie::time::error::ComponentRange, http::StatusCode, ResponseError};
+
+/// This error can occur during login attempts.
+#[derive(Debug)]
+pub struct LoginError(SessionInsertError);
+
+impl std::fmt::Display for LoginError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+impl std::error::Error for LoginError {}
```

Comment:
We must implement the `source` method and return the wrapped error.

Reply:
Oh! I didn't realize. Easy enough to do  :)